log/log.go

```diff
@@ -45,6 +48,20 @@ func NewLogger(config config.Log) (*zap.Logger, error) {
 	return logger, nil
 }
 
+func GRPCRequestToLogFields(info *grpc.StreamServerInfo, start time.Time, err error) []zapcore.Field {
```

Comment:
__[golint]__
* [golint] exported function GRPCRequestToLogFields should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->